var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -42,7 +42,7 @@ class Fftw(Package):
     variant('float', default=True, description='Produces a single precision version of the library')
     variant('long_double', default=True, description='Produces a long double precision version of the library')
     variant('quad', default=False, description='Produces a quad precision version of the library (works only with GCC and libquadmath)')
-
+    variant('openmp', default=True, description="Enable OpenMP support.")
```

Comment:
i would put it to `False` by default. Interesting question is if `FFTW` is more cleaver than openblas in that they do support OpenMP with non-Apple's clang? If so, one could get away with something like
```
if not spec.satisfies('%clang@6.0.0-apple:7.3.0-apple'):
   options.append('--enable-openmp')
```
to filter out Apple's clang and remove the variant completely.